### DISCUSSION TITLE
Toolbar: download and rotation

### DISCUSSION THREAD

**Discussion by Pluxx** [fancyapps/ui#290] [Q&A] [answered]
Hi, is it possible to enable a download button for the viewer and a rotate button with fancybox on the toolbar?

The v4 options don't seem to have a button control section anymore.

Trying to evaluate if this lib can be used for my project. Not much of a js guy.

**Comment by fancyapps** [comment, marked as answer]
Hi,

Here is an example on how to enable download button - https://fancyapps.com/docs/ui/fancybox/plugins/toolbar - but rotate/flip will be available starting from v5.
